source/frontend/src/features/properties/map/compensation/update/UpdateCompensationRequisitionForm.tsx

```diff
@@ -52,6 +76,31 @@ const UpdateCompensationRequisitionForm: React.FC<CompensationRequisitionFormPro
     }
   };
 
+  const onFinancialActivitiesUpdated = (values: CompensationRequisitionFormModel) => {
```

Comment:
Let me check with Priyanka on this, but I believe that the ETLs are not entering data into the PIMS_PAYEE_CHEQUE, they are expecting that the system will just use the summed values from the financials. I'm partial to that as well since it reduces the risk of the calculations and the value in PIMS_PAYEE_CHEQUE gets out of sync.

Reply:
Ok, I'm wrong, your pattern here should work fine.